conda_config.py

```diff
@@ -0,0 +1,38 @@
+import yaml, argparse, os
+
+
+class CondaDumper(yaml.Dumper):
+
+    def increase_indent(self, flow=False, indentless=False):
+        return super(CondaDumper, self).increase_indent(flow, False)
+
+
+def write_conda_recipe(version):
+    """
+    Writes the EvalML recipe to build a conda package based on the version
+    Args:
+        version: The version of EvalML we are building with this feedstock
+
+    Returns:
+        None: Side effect of overwriting the existing meta.yaml in the feedstock
+    """
+
+    recipe_file_path = os.path.join(os.path.join(os.path.join(os.getcwd(), 'evalml-core-feedstock'), 'recipe'), 'meta'
+                                                                                                                '.yaml')
+    with open(recipe_file_path, 'rb') as config_file:
+        # Toss out the first line that declares the version since its not supported YAML syntax
+        next(config_file)
+        config = yaml.safe_load(config_file)
+        recipe_path = f'../feedstock_root/evalml'
```

Comment:
@ryan-s could you please [use `pathlib.Path`](https://docs.python.org/3/library/pathlib.html) here instead? For hypothetical windows support haha

```python
recipe_path = str(pathlib.Path('..', 'feedstock_root', 'evalml'))
```

Also what is this path? Like, this is intended to be the path on the docker container? A comment explaining this would be super helpful to future readers.